src/Adapter/StockManager.php

```diff
@@ -28,15 +28,14 @@
 
 use Db;
 use PrestaShop\PrestaShop\Adapter\Shop\Context as ShopAdapter;
-use PrestaShopBundle\Service\DataProvider\StockInterface;
 use StockAvailable;
 
 /**
  * Data provider for new Architecture, about Product stocks.
  *
  * This class will provide data from DB / ORM about Product stocks.
  */
-class StockManager implements StockInterface
+class StockManager
```

Comment:
?

Reply:
The interface was removed because it's not used anywhere The method it's supposed to implement is still present because it's the only service that implements it but even this method has been deprecated and will be removed in 10.0

Reply:
👍